Dockerfile.prod

```diff
@@ -33,7 +33,16 @@ RUN bundle install --without development test
 
 COPY . /usr/src/app/
 
+# For ActiveAdmin assets
 # https://github.com/rails/rails/issues/32947#issuecomment-511532631
 RUN SECRET_KEY_BASE=dummy bin/rails assets:precompile
 
+# For puma server
 RUN mkdir -p tmp/pids
+
+# Build vue files
+COPY vue/package*.json vue/yarn.lock /usr/src/app/vue/
```

Comment:
Also, should you need to copy anything else if you already copied `.`?

Reply:
Oh good point, I was trying to take advantage of the caching layer and copy the package/yarn files first, but I'm doing it after I copy everything so it doesn't make any sense. But I don't think anything is cached anyway with this Heroku setup and that it runs everything from scratch every time you deploy anyway, so it shouldn't matter.